bin/test/game_spec.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/game'
+
+describe Game do
+ 
+  before(:all) do
+    @player1 = Player.new("david", "X")
```

Comment:
You can use `let` syntax here. Ref: https://dzone.com/articles/rspec-let-vs-before